pkg/ts/catalog/chart_catalog.go

```diff
@@ -853,6 +853,10 @@ var charts = []sectionDescription{
 				Title:   "Keys with GC'able Data",
 				Metrics: []string{"queue.gc.info.numkeysaffected"},
 			},
+			{
+				Title:   "Range tombstones GC'able",
```

Comment:
Let's use the more general "Range keys" here, since that's what we refer to them as in e.g. MVCC stats.

Reply:
I'd use "Range Keys with GC'able Data". We're GCing their metadata.

>We don't plan to use range keys with values any time soon... or not?

No, not in the near-term. The next use-cases that come to mind are ranged intents (to make MVCC range tombstones transactional) and keyspace locking.